### ISSUE TITLE
Hero Video: Play/Pause button doesn't have accessible name

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#65]
The pause/play button does not contain any text to describe it's purpose to users. 	

Include a title or aria label on the button that describes its purpose.

**Comment by syarwick23** [comment]
Play/pause is accessible to screen readers!
